examples/trials/mnist-tfv2/config_remote.yml

```diff
@@ -24,9 +24,5 @@ machineList:
   - ip: ${replace_to_your_remote_machine_ip}
     username: ${replace_to_your_remote_machine_username}
     sshKeyPath: ${replace_to_your_remote_machine_sshKeyPath}
-    # Pre-command will be executed before the remote machine executes other commands.
     # Below is an example of specifying python environment.
-    # If you want to execute multiple commands, please use "&&" to connect them.
-    # preCommand: source ${replace_to_absolute_path_recommended_here}/bin/activate
-    # preCommand: source ${replace_to_conda_path}/bin/activate ${replace_to_conda_env_name}
-    preCommand: export PATH=${replace_to_python_environment_path_in_your_remote_machine}:$PATH
+    pythonPath: ${replace_to_python_environment_path_in_your_remote_machine}
```

Comment:
better to add comments to show concrete path examples here

Reply:
Done.

Reply:
The doc said "The path will insert to PATH before launching each trial". Is `/usr/bin/python3.7` supposed to be a directory?

Reply:
It is a directory, "python3.7" is a folder name..I will rename it to "yourPythonPath" to avoid the misunderstanding.

Reply:
Then I suggest `/opt/python3.7/bin` for Linux example.

Reply:
Done.